hooks/utils.py

```diff
@@ -346,6 +348,8 @@ def write_builtin_server_startup(
         context['tests_root'] = os.path.join(JUJU_GUI_DIR, 'test', '')
     render_to_file(
         'guiserver.conf.template', context, GUISERVER_INIT_PATH)
+    render_to_file(
+        'guiserver.service.template', context, GUISERVER_SERVICE_PATH)
```

Comment:
So we create both scripts and can rely on the correct one being used based on series, right?